src/components/organisms/NotePageToolbar.tsx

```diff
@@ -171,21 +171,6 @@ const NotePageToolbar = ({ storage, note }: NotePageToolbarProps) => {
     },
     [storage.id, note, updateNote]
   )
-
-  const getAttachmentData = useCallback(
```

Comment:
Please revoke this change. All attachments must be retrieved from this method.

Reply:
If we really need to access the map, then we can introduce a new method to retrieve it. But please avoid introducing the method if possible.

Reply:
Umm, I am confused by this comments.

So in essence, we need to have attachments data, this means attachmentMap is an variable wee need access to.
This can be done either via sending it in a method and using it directly, or (as previously coded) using the `getAttachmentData` method, which was retriving attachment data for a particular `src` value.

From first comment, you said we should not need this method maybe, so I removed it in this refactor, and used attachmentMap directly. 

How would new method to retrieve it be used like? do you mean a method to retrieve attachmentMap, or attachment data? or something else?

I cannot really think of a way to get needed attachments without using attachmentData or some method to retrtieve it for particular found `src` in content.

Maybe we could somehow preprocess content and fetch attachments needed (similar to how it is done in export now)
and send attachments needed to export? Or in the preprocess step embedd needed changes, thus not needing to send the attachments at all - and send modified content?